client/scripts/views/modals/manage_community_modal/chain_metadata_management_table.ts

```diff
@@ -48,107 +56,162 @@ const ChainMetadataManagementTable: m.Component<IChainOrCommMetadataManagementAt
     vnode.state.symbol = vnode.attrs.chain.symbol;
     vnode.state.snapshot = vnode.attrs.chain.snapshot;
   },
-  view: (vnode) => {
+  view: (vnode: any) => {
```

Comment:
since when do we type `vnode`?

Reply:
we should then type it what it is, rather than "any"

Reply:
Doesn't work lol, I tried/I wish. We could just disable the rule, or figure out big picture why it isn't working.